CRM/Utils/PDF/Utils.php

```diff
@@ -98,23 +90,23 @@ public static function html2pdf(&$text, $fileName = 'civicrm.pdf', $output = FAL
   <head>
     <meta http-equiv=\"Content-Type\" content=\"text/html; charset=utf-8\"/>
     <style>@page { margin: {$t}{$metric} {$r}{$metric} {$b}{$metric} {$l}{$metric}; }</style>
-    <style type=\"text/css\">@import url({$config->userFrameworkResourceURL}css/print.css);</style>
+    <style type=\"text/css\">@import url(" . CRM_Core_Config::singleton()->userFrameworkResourceURL . "css/print.css);</style>
```

Comment:
What's the reason for this change? Looks to me like the `$config` variable has been defined and should work fine here.

Reply:
yeah it does but it doesn't really need to be defined since the other place it's being used should really use a setting - using config instead of setting is legacy - and it should really be in a separate fn anyway so no need to declare $config after all